frontend/src/translations/result/br.json

```diff
@@ -33,6 +33,8 @@
   "payment.creditcard": "Cartão de crédito",
   "payment.paypal.title": "Fazer um pagamento com o Paypal",
   "payment.paypal.description": "Lembre-se de que a pessoa a quem essa tarefa for atribuída receberá o pagamento obrigatoriamente pelo Paypal",
+  "payment.paypal.warning": "Pagamentos realizados pelo Paypal esta sujeito a cobranças da plataforma.",
```

Comment:
We don't need to add translations strings in this file, it's a generated file by other tool